src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,51 @@
+package model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(String operation, String fruit, int quantity) {
+        this.operation = convertOperation(operation);
```

Comment:
you shouldn't call methods here
```suggestion
        this.operation = operation;
```